qiskit/quantum_info/states/random.py

```diff
@@ -54,7 +55,12 @@ def random_statevector(dims, seed=None):
     return Statevector(np.sqrt(x / sumx) * np.exp(1j * phases), dims=dims)
 
 
-def random_density_matrix(dims, rank=None, method="Hilbert-Schmidt", seed=None):
+def random_density_matrix(
+    dims: int | tuple,
+    rank: int | None = None,
+    method: str = "Hilbert-Schmidt",
```

Comment:
```suggestion
    method: Literal[ "Hilbert-Schmidt", "Bures"] = "Hilbert-Schmidt",
```
is better. (It is necessary tp add import.)